workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/PublisherDashboardRouter.tsx

```diff
@@ -90,7 +90,10 @@ const PublisherDashboardRouter = () => {
 					<Route element={<Accounts />} path="accounts" />
 					<Route element={<Members />} path="members" />
 					<Route element={<Projects />} path="projects" />
-					<Route element={<Solutions />} path="solutions" />
+
+					<Route path="solutions">
+						<Route element={<Solutions />} index />
+					</Route>
```

Comment:
@joanagrego changed this line here: https://github.com/is-solutions-delivery/liferay-portal/pull/5720/files#diff-dadf03fa0abb5395bc35fc8911e4c016d9fa1e994d9478e000b36d4bc20d104bR95

Talk to her who should send this code to avoid conflicts